src/routes/InventoryTable.js

```diff
@@ -50,11 +50,26 @@ class Inventory extends Component {
         this.updateEntities = updateEntities;
 
         this.setState({
-            ConnectedInventory: InventoryTable
+            ConnectedInventory: InventoryTable,
+            removeListener
+        });
+    }
+
+    onRefresh({ filters }) {
+        this.setState({
+            filters
         });
     }
 
-    onRefresh() {
+    componentWillUnmount() {
+        this.inventory && this.state.removeListener();
+    }
+
+    async onSelect() {
+        const { filters } = this.state;
+        const results = await getAllEntities({ filters });
+        // eslint-disable-next-line no-console
+        console.log(results, 'gd');
```

Comment:
do you want to keep this here?

Reply:
It was for testing download button, it's no longer there.